functions/Get-DbaClusterSqlInstance.ps1

```diff
@@ -0,0 +1,84 @@
+function Get-DbaClusterSqlInstance
+{
+<#
+.SYNOPSIS
+Returns the registered SQL Server Instances on a windows Server cluster
+	
+.DESCRIPTION
+By default, this command returns for each Cluster passed in:
+Pulls Cluster details via WMI/CIM so doesn't require Failover Cluster PowerShell module to be installed.
+Multiple clusters can be piped in.
+Returns ALL instance registered on cluster. Does not check that they are viable, correctly installed or currenly running. 
+	
+.PARAMETER Cluster
```

Comment:
You can do, I was trying to be accurate.

Tbh, if the cluster is setup properly then just using a node name will work (does with my testing, and my opinion is that mine are setup correctly ;) )